src/main/java/io/aiven/kafka/connect/opensearch/OpensearchSinkConnectorConfig.java

```diff
@@ -397,13 +404,38 @@ private static void addConversionConfigs(final ConfigDef configDef) {
                 group,
                 ++order,
                 Width.SHORT,
-                "Behavior on document's version conflict (optimistic locking)");
+                "Behavior on document's version conflict (optimistic locking)"
+        ).define(
+                ERRORS_TOLERANCE_CONFIG,
+                Type.STRING,
+                BulkProcessor.ErrorToleranceType.DEFAULT.toString(),
+                BulkProcessor.ErrorToleranceType.VALIDATOR,
+                Importance.LOW,
+                ERRORS_TOLERANCE_DOC,
+                group,
+                ++order,
+                Width.SHORT,
+                "Error Tolerance");
```

Comment:
We should not be redefining these properties in the connector config; they should be left up to the Connect runtime.